test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -10966,21 +10957,18 @@ def add_server_dyn_res(self, custom_resource, script_body=None, file=None,
             script_body = f.readlines()
             f.close()
         else:
-            (fd, file) = self.du.mkstemp(prefix='PtlPbsSchedConfig')
-            f = open(file, "w")
-            f.write(script_body)
-            f.close()
-            os.close(fd)
+            fn = self.du.create_temp_file(prefix='PtlPbsSchedConfig',
+                                          body=script_body)
 
-        self.server_dyn_res = file
-        self.logger.info(self.logprefix + "adding server dyn res " + file)
+        self.server_dyn_res = fn
```

Comment:
Here fn will be undefined in file is not None (aka if block).

Also is it possible to rename argument 'file' to something else like input_file? because file is keyword in python.